Query Management and Configuration Clean-up for Enterprise 1.9 (#2733)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

Environment variable: `INFLUXDB_TLS_MAX_VERSION`

## InfluxQL query management settings

### `[coordinator]`

This section contains configuration settings for query management.
For more on managing queries, see [Query Management](/enterprise_influxdb/v1.9/troubleshooting/query_management/).

#### `write-timeout = "10s"`

> The duration a write request waits until a "timeout" error is returned to the caller. The default value is 10 seconds.

Reviewer: ```suggestion
Duration a write request waits until a "timeout" error is returned to the caller. Default is `10s` (10 seconds).
```

Environment variable: `INFLUXDB_COORDINATOR_WRITE_TIMEOUT`

#### `max-concurrent-queries = 0`

> The maximum number of running queries allowed on your instance.
> The default setting (`0`) allows for an unlimited number of queries.

Reviewer: ```suggestion
Maximum number of concurrently running queries allowed. `0` means unlimited. Default is `0`.
```

Environment variable: `INFLUXDB_COORDINATOR_MAX_CONCURRENT_QUERIES`

#### `query-timeout = "0s"`

> The maximum duration that a query is allowed to execute before InfluxDB
> kills the query.
> The default setting (`0`) allows queries to run with no time restrictions.
> This setting is a [duration](#configuration-overview).

Reviewer: ```suggestion
Maximum duration a query is allowed to execute before InfluxDB kills the query.
`0s` allows queries to run with no time restrictions.
Default is `0s`.
```

Environment variable: `INFLUXDB_COORDINATOR_QUERY_TIMEOUT`

#### `log-queries-after = "0s"`

> The maximum duration that a query can until InfluxDB logs the query with a
> `Detected slow query` message.
> The default setting (`"0"`) will never tell InfluxDB to log the query.
> This setting is a [duration](#configuration-overview).

Reviewer: ```suggestion
Maximum duration a query can run until InfluxDB logs the query with a
`Detected slow query` message.
`0s` disables slow query logging.
Default is `0s`.
```

Environment variable: `INFLUXDB_COORDINATOR_LOG_QUERIES_AFTER`

#### `max-select-point = 0`

> The maximum number of [points](/enterprise_influxdb/v1.9/concepts/glossary/#point) that a
> `SELECT` statement can process.
> The default setting (`0`) allows the `SELECT` statement to process an unlimited
> number of points.

Reviewer: ```suggestion
Maximum number of [points](/enterprise_influxdb/v1.9/concepts/glossary/#point) a
`SELECT` statement can process.
`0` lets the `SELECT` statement to process an unlimited number of points.
Default is `0`.
```

Environment variable: `INFLUXDB_COORDINATOR_MAX_SELECT_POINT`

#### `max-select-series = 0`

> The maximum number of [series](/enterprise_influxdb/v1.9/concepts/glossary/#series) that a
> `SELECT` statement can process.
> The default setting (`0`) allows the `SELECT` statement to process an unlimited
> number of series.

Reviewer: ```suggestion
Maximum number of [series](/enterprise_influxdb/v1.9/concepts/glossary/#series) a
`SELECT` statement can process.
`0` lets the `SELECT` statement to process an unlimited number of series.
Default is `0`.
```

Environment variable: `INFLUXDB_COORDINATOR_MAX_SELECT_SERIES`

#### `max-select-buckets = 0`

> The maximum number of `GROUP BY time()` buckets that a query can process.
> The default setting (`0`) allows a query to process an unlimited number of
> buckets.

Reviewer: ```suggestion
Maximum number of `GROUP BY time()` buckets a query can process.
`0` allows a query to process an unlimited number of buckets.
Default is `0`.
```

Environment variable: `INFLUXDB_COORDINATOR_MAX_SELECT_BUCKETS`

## Flux query management settings

### `[flux-controller]`

> This section contains configuration settings for query management.

Reviewer: ```suggestion
This section contains configuration settings for Flux query management.
```
